docs/eeeoh.md

```diff
@@ -0,0 +1,472 @@
+# eeeoh
+
+This page contains usage guidance for SEEK's proprietary logging solution.
+
+SEEKers can visit [Backstage] for more information.
+These optional features are not relevant to applications that run outside of SEEK's standard workload hosting environments.
+
+## Getting started
+
+To opt in:
+
+```typescript
+import { createLogger } from '@seek/logger';
+
+const base = {
+  env: 'production',
+  service: 'my-component-name',
+  version: 'abcdefa.123',
+} as const;
+
+const logger = createLogger({
+  base,
+  eeeoh: { datadog: 'tin' },
+});
+```
+
+In practice, you'd likely read `base` values from environment variables:
+
+```typescript
+import { Eeeoh } from '@seek/logger';
+
+// `Env` is like `process.env` but throws an error if the variable is not set.
+// We recommend failing fast over silently continuing in a misconfigured state.
+import { Env } from 'skuba-dive';
+
+const base = {
+  env: Env.oneOf(Eeeoh.envs)('DD_ENV'),
+  service: Env.string('SERVICE'),
+  version: Env.string('VERSION'),
+} as const;
+
+const logger = createLogger({
+  base,
+  eeeoh: { datadog: 'tin' },
+});
+```
+
+Note that `@seek/logger` uses simplified syntax for its configuration options.
+Logs are internally transformed to the output format expected by eeeoh:
+
+```json
+{
+  "ddsource": "nodejs",
+  "ddtags": "env:production,version:abcdefa.123",
+  "eeeoh": {
+    "logs": {
+      "datadog": {
+        "enabled": true,
+        "tier": "tin"
+      }
+    }
+  },
+  "env": "production",
+  "service": "my-component-name",
+  "version": "abcdefa.123"
+}
+```
+
+## Base attributes
+
+eeeoh prescribes [Datadog unified service tagging] as a baseline.
+Opting in mandates these attributes in the `base` configuration option:
+
+```diff
+import { createLogger } from '@seek/logger';
+import { Env } from 'skuba-dive';
+
+createLogger({
+  base: {
+-   environment: 'prod',
++   env: 'production',
+
+-   app: 'my-app',
+-   name: 'my-app',
++   service: 'my-component-name',
+
+    version: Env.string('VERSION'),
+  },
+});
+```
+
+`env` is one of the following values for internal consistency:
+
+- `development`
+- `production`
+- `sandbox`
+- `test`
+
+`service` is the name of the [component], or the service name override on a [deployment] of the component.
+
+`version` is a unique identifier for the current deployment.
+Incorporate the commit hash and build number;
+see subsequent sections for implementation guidance.
+
+`@seek/logger` currently auto-generates `ddsource` and `ddtags`.
+Reach out if you have a need to set custom values for these attributes.
```

Comment:
> [I think the values that we include in the ddtags field are automatically included in the facets side panel. Could be worth auto including some and/or providing guidance on what people should include?](https://github.com/seek-oss/logger/pull/184#issuecomment-2998680393)

I'm proposing that we don't recommend anything custom at the moment given implications on indexing cost & query performance. Any objections?